### ISSUE TITLE
MA0003 should not be friggered for ValueTask.FromResult

### ISSUE THREAD

**Issue by hankovich** [meziantou/Meziantou.Analyzer#584]
### Version of the Meziantou.Analyzer NuGet package

2.0.74

### Rule Identifier

MA0003 

### Target Framework

net8.0

### C# Language version

C# 12

### Description

MA0003 successfully excludes `Task.FromResult`, but is triggered for `ValueTask.FromResult`

### Reproduction Steps

Minimal code:

```c#
_ = ValueTask.FromResult<ReadOnlyMemory<byte>?>(null);;
```

**Comment by meziantou** [comment]
It should be fixed in the next release!
